doc/readme/DEVELOPING.md

```diff
@@ -156,9 +155,9 @@ For example
 
 #### Adding tests
 We aim to write clear, minimal, yet exhaustive tests of the basic building blocks in Shogun.
-Whenever you send us C++ code, we will ask for a unit test of it.
+Whenever you send us the C++ code, we will ask for a unit test of it.
```

Comment:
As said before, there is no need to change us such changes. This doesnt improve the text.